deploy/operator.yaml

```diff
@@ -34,8 +34,7 @@ spec:
       containers:
         - name: kata-operator
           # Replace this with the built image name
-          #image: quay.io/isolatedcontainers/kata-operator:v1.0
-          image: quay.io/isolatedcontainers/kata-operator:v1.0
+          image: quay.io/isolatedcontainers/kata-operator:v2.0
```

Comment:
Should we add a config map to specify that image instead of hard-coding it in a source code?

Reply:
yes, but I'd do it as a follow-on PR?